scripts/hak/index.js

```diff
@@ -133,4 +133,4 @@ async function main() {
     }
 }
 
-main();
+main().catch(process.exit(1));
```

Comment:
should this not be `() => process.exit(1)`

Reply:
oops - shows what you get for testing only that it exits on failure.